### PR TITLE
1030: Incomplete data privacy

### LINKED ISSUE #1030: [Bavaria] Incomplete data privacy policy
**Describe the bug**
In the data privacy policy the content of the section "Weitergabe der Daten an externe Dritte" is missing.
This sections will be extended for regional specific content for "beantragen" but in the general "base" policy there should be a text included

**To Reproduce**
1. Go to https://bayern.ehrenamtskarte.app/data-privacy-policy
2. Scroll down to "Weitergabe der Daten an externe Dritte"
3. Check for no content

**Expected behavior**
If the data privacy policy is used as a general policy and not extended by regional content it should be displayed:
"Es erfolgt keine Weitergabe der Daten an Dritte."
Additionally this information should be shown if no additional regional content is provided

### PR DESCRIPTION
**Pull request by f1sh1918**
resolves #1030 

- If no region specific policy is provided show additional base text
- show additional base text for general data privacy policy (linked in app)

### CODE REVIEW

File: administration/src/project-configs/bayern/dataPrivacyBase.tsx [hunk 1]
```diff
@@ -449,11 +449,16 @@ export const DataPrivacyBaseText = () => (
       Informationen, die Sie unverschlüsselt per E-Mail an uns senden können, möglicherweise auf dem Übertragungsweg von
       Dritten gelesen werden.{' '}
     </p>
-    <p style={{ marginTop: '0pt', marginBottom: '8pt' }}>
+    <div>
```
Comment (michael-markl) [inline administration/src/project-configs/bayern/dataPrivacyBase.tsx:452]: Should we readd the margin bottom here?

File: administration/src/project-configs/bayern/dataPrivacyBase.tsx [hunk 2]
```diff
@@ -449,11 +449,16 @@ export const DataPrivacyBaseText = () => (
...
         <b>Weitergabe der Daten an externe Dritte</b>
       </span>
-    </p>
-    <p style={{ marginTop: '0pt', marginBottom: '8pt' }} />
+    </div>
+  </div>
+)
+
+export const DataPrivacyAdditionalBaseText = () => (
+  <div>
```
Comment (michael-markl) [inline administration/src/project-configs/bayern/dataPrivacyBase.tsx:461]: Should we make this a p? Or is some manual margin necessary?

**Review by michael-markl — APPROVED** [review]
Did not test it.

File: administration/src/project-configs/bayern/dataPrivacyBase.tsx (hunk 1 above)
Comment (f1sh1918) [reply administration/src/project-configs/bayern/dataPrivacyBase.tsx:452]: hm its very inconsistent in the template. the sections directly above do not use the margin after the headline. so it looks weird imo

File: administration/src/project-configs/bayern/dataPrivacyBase.tsx (hunk 2 above)
Comment (f1sh1918) [reply administration/src/project-configs/bayern/dataPrivacyBase.tsx:461]: we could add some bottom margin, but on the other hand if the region specific text gonna be added there won't be any margin neither. I think it's fine as it is.

image

**Pull request merged**